src/PrestaShopBundle/EventListener/ModuleActivatedListener.php

```diff
@@ -57,7 +58,7 @@ class ModuleActivatedListener
     private $translator;
 
     /**
-     * @var SessionInterface
+     * @var Session
```

Comment:
🤔  sure about that ? usually we prefer to type-hint the Interface rather than the implementation

Reply:
You're right. I checked the `src/PrestaShopBundle/Resources/config/services/bundle/event_listener.yml` and it's well `Session`. I change the __construct. Is it a BC ?

Reply:
Ping @matks

Reply:
Yes it's the BC (if someone else used another class implementing SessionInterface, with this code change it will fail for him).

And we _want_ to require the interface, not the implementation. That is good practice 🤔

Reply:
The problem is that SessionInterface doesn't declare `getFlashBag` method used at l.143.

Reply:
Could we use `public function getBag($name);` instead, which belongs to SessionInterface method

Reply:
@matks I fixed without BC ;)